### ISSUE TITLE
Multiple inputs for INC tensorflow dataloader

### ISSUE THREAD

**Issue by zhentaocc** [intel/neural-compressor#52]
I'm using INC to quantize a keras model, with input format as `((x1, x2,), y1)`.  
I debugged to https://github.com/intel/neural-compressor/blob/master/neural_compressor/experimental/data/dataloaders/tensorflow_dataloader.py#L91-L97, and seems it only supports one single input for now. But I think it's a simple to repeat what you process on label to enable multiple inputs, correct?

**Comment by zhentaocc** [comment]
I tried this on my network and it works:
```python
                if isinstance(samples,tf.Tensor):
                    samples.append(iter_inputs.numpy())
                else:
                    samples.append([np.array(i) for i in iter_inputs])
                if isinstance(iter_labels,tf.Tensor):
                    samples.append(iter_labels.numpy())
                else:
                    samples.append([np.array(l) for l in iter_labels])
```

**Comment by zhentaocc** [comment]
A minimum script for you to reproduce:
```python
from neural_compressor.experimental.data.dataloaders import DATALOADERS
import numpy as np
import tensorflow as tf
x = tf.data.Dataset.from_tensor_slices((np.random.random(20), np.random.random(20)))
y = tf.data.Dataset.from_tensor_slices(np.random.random(20))
dataset = tf.data.Dataset.zip((x, y))
dataloader = DATALOADERS['tensorflow'](dataset)
for i, (x, y) in enumerate(dataloader):
    print(x, y)
```

**Comment by Zhiwei35** [comment]
already generate patch and send to @zhentaocc

**Comment by zhentaocc** [comment]
Could you inform me of the status if you fixed this and which version?

**Comment by chensuyue** [comment]
Please check with v1.11 release which will be delivered on 4/15.

**Comment by zhentaocc** [comment]
> Please check with v1.11 release which will be delivered on 4/15.

thx
